app/popup.tsx

```diff
@@ -103,6 +103,13 @@ chrome.tabs.query(queryParams, tabs => {
   const { id, url } = tab;
   if (typeof id === "undefined" || typeof url === "undefined") return;
 
+  MessageService.getCount(id, (count) => {
+  chrome.browserAction.setBadgeText({
+      text: `${count}`,
```

Comment:
no need for the string interpolation.
just use `text: count`

Reply:
We have declared count to be of number type https://github.com/code-mancers/interceptor/blob/4cda610db7930ff9c4042e055adc5799615f118c/app/background.ts#L8 , we need interpolation I guess.